### ISSUE TITLE
Unable to resolve module `AccessibilityInfo`

### ISSUE THREAD

**Issue by jgautheron** [solinor/react-native-bluetooth-status#5]
When installing this package, the `AccessibilityInfo` keeps popping because of conflicting `react-native` versions, read more about this problem here:
https://github.com/facebook/react-native/issues/14209

**Comment by smartdevdom** [comment]
I have same error

**Comment by eduplus** [comment]
Made some changes to the version numberings to fix these issues, npm has 1.1.1 now, let me know if you still encounter problems.

**Comment by jgautheron** [comment]
@eduplus coud you remove the `react` and `react-native` from `dependencies`? any different version will cause issues.
See similar issues here: https://github.com/mientjan/react-native-markdown-renderer/issues/5
